test/nightly/azureResourceMongoDB.test.ts

```diff
@@ -75,12 +83,18 @@ suite('MongoDB action', async function (this: ISuiteCallbackContext): Promise<vo
         assert.ifError(accountExists);
     });
 
-    async function getMongoClient(): Promise<MongoClient> {
+    async function getMongoClient(resourceName: string): Promise<MongoClient> {
```

Comment:
I think it'd be better to use a regexp to find the account name.  If we change the label name (the MongoDB part), it would break the tests.